src/python/WMQuality/WebTools/RESTClientAPI.py

```diff
@@ -58,10 +58,12 @@ def makeRequest(url, values=None, verb='GET', accept="text/plain",
     conn.connect()
     conn.request(verb, uri, data, headers)
     response = conn.getresponse()
-
     data = response.read()
     conn.close()
     cType = response.getheader('content-type').split(';')[0]
+    # data returned could be something a json like: b'"foo"', so we need to properly load it
+    #if '/json' in accept:
+    #    data = json.loads(data)
```

Comment:
Are these comments used as a memo in case we face problems in the future?

Reply:
Exactly! It took me a couple of hours to pin point this problem. Given that this code isn't deployed in our production systems, I think it would be good to leave it around :-P